java/pom.xml

```diff
@@ -155,8 +168,12 @@
                 </executions>
                 <configuration>
                     <redirectTestOutputToFile>true</redirectTestOutputToFile>
-                    <forkMode>once</forkMode>
+
+                    <!-- It's important to run tests serially because we are doing timing tests in some of the tests. -->
```

Comment:
You can check it out at the maven [surefire plugin docs](https://maven.apache.org/surefire/maven-surefire-plugin/examples/fork-options-and-parallel-execution.html).  Search for "**Forked Test Execution**".  

> The parameter forkCount defines the maximum number of JVM processes that maven-surefire-plugin will spawn concurrently to execute the tests. It supports the same syntax as `-T` in maven-core: if you terminate the value with a 'C', that value will be multiplied with the number of available CPU cores in your system. For example `forkCount=2.5C` on a Quad-Core system will result in forking up to ten concurrent JVM processes that execute tests.
>
> The parameter reuseForks is used to define whether to terminate the spawned process after one test class and to create a new process for the next test in line (`reuseForks=false`), or whether to reuse the processes to execute the next tests (`reuseForks=true`).